warehouse/templates/email/primary-email-change.body.txt

```diff
@@ -0,0 +1,5 @@
+Someone, perhaps you, has changed the primary email for your PyPI account
+'{{ username }}'.
```

Comment:
I think it'd be nice here if this email included the emails it changed from/to. So something like 

```
The primary email for your PyPI account:

  '{{ username }}'

been changed from this address:

  {{ old_email }}

to this address:

  {{ new_email }}

If you did not make this change, you can reply to this email directly to
communicate with the PyPI administrators.
```